ivy/functional/frontends/tensorflow/math.py

```diff
@@ -361,3 +361,8 @@ def truediv(x, y, name="truediv"):
     elif x_dtype in [ivy.int32, ivy.uint32, ivy.int64, ivy.uint64]:
         return ivy.divide(ivy.astype(x, ivy.float64), ivy.astype(y, ivy.float64))
     return ivy.divide(x, y)
+
+
+
+
```

Comment:
I don't think these empty lines are needed. They might cause a lint error. Only one empty line at the end of each file.